### PR TITLE
[openshift/cert-manager-operator] Add jobs for cert-manager-1.10,1.11 custom branches

### PR DESCRIPTION
**Pull request by swghosh**
Signed-off-by: Swarup Ghosh <swghosh@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.11.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+- as: e2e-operator-upgrade
+  optional: true
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-cert-manager-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1
+      OO_INSTALL_NAMESPACE: cert-manager-operator
+      OO_LATEST_CSV: cert-manager-operator.v1.11.1
```
Comment (thejasn) [inline ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.11.yaml:108]: 1.11.1 or 1.11.4?

Comment (swghosh) [reply ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.11.yaml:108]: Trying to see if rehearsals pass first.
Ideally, would be v1.11.4: yes (once most rehearsal tests pass).

Comment (swghosh) [reply ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.11.yaml:109]: Followed up in https://github.com/openshift/release/pull/40954.

**Pull request merged**